apps/explorer/src/pages/home/Home.tsx

```diff
@@ -19,7 +19,7 @@ import { SuiTokenCard } from '~/components/SuiTokenCard';
 import { Card } from '~/ui/Card';
 import { Tab, TabGroup, TabList, TabPanel, TabPanels } from '~/ui/Tabs';
 
-const NodeMap = lazy(() => import('../../components/node-map'));
```

Comment:
can you please remove this while you're at this? Don't want duplicated code and it's harder to review diffs that are all additive

Reply:
@mystie711 are we trying to replace the node map on the validators page as well? or should we keep that as is